dashboard/app/jobs/cap/sections_warning_email_sender.rb

```diff
@@ -0,0 +1,56 @@
+require 'metrics/events'
+require 'services/child_account'
+
+module CAP
+  # Sends an email to a teacher warning them about which sections have students
+  # currently being affected by our Child Account Policy(CAP).
+  class SectionsWarningEmailSender < ApplicationJob
+    rescue_from StandardError, with: :report_exception
+
+    def perform(teacher_id:, section_ids:)
+      return unless teacher_id
```

Comment:
I believe that if we try to provide `nil` as a teacher ID somewhere in the code, it's something we want to be aware of rather than ignore such a mistake:
```suggestion
      raise ArgumentError, 'teacher_id must be present' unless teacher_id
```